### ISSUE TITLE
Error not thrown, but debug logging is output

### ISSUE THREAD

**Issue by klobetime** [zorkow/speech-rule-engine#550]
I came across this due to an accident, but it sure seems strange. When passing wildly invalid MathML to `toSpeech()` I saw some error messages to stdout but the call returned successfully with an empty string!

## package.json
```json
{
  "name": "example",
  "dependencies": {
    "speech-rule-engine": "=3.2.0"
  }
}
```
## server.js
```node
'use strict';

const sre = require('speech-rule-engine');

sre.setupEngine({
    domain: 'mathspeak'
});

sre.engineSetup();

const speech = sre.toSpeech('<math><mrow><msup><mn>3')
console.log('speech=', speech);
```

## Result
```
$ npm start

> example@ start ...
> node server.js

[xmldom warning]	unclosed xml attribute 
@#[line:1,col:1]
[xmldom warning]	unclosed xml attribute 
@#[line:1,col:7]
[xmldom warning]	unclosed xml attribute 
@#[line:1,col:13]
[xmldom warning]	unclosed xml attribute 
@#[line:1,col:19]
speech= 
```

Clearly I'm not expecting any magic to complete the MathML or anything, but I would like to be able to detect some sort of error without monitoring stdout. And the fact that something goes to stdout seems odd regardless. Hopefully I just have this misconfigured!

**Comment by zorkow** [comment]
The problem is that the error is flagged by `xmldom` as a warning so can not be caught in SRE.
I'll have a look if we can expose that somehow smarter.

**Comment by klobetime** [comment]
Another weird one for you along these same lines. Using the math '<math><mtext mathvariant-"normal">Texas</mtext></math>' (stumbled on this due to the typo in the mathvariant attribute) with v5.0.0-alpha.8 shows this on the stderr but completes successfully with empty speech:

```
[xmldom warning]	attribute value must after "="
@#[line:1,col:7]
Something went wrong computing speech.
```

Using v4.1.2 has the same result, but with an additional TypeError stack trace being sent to stdout.

NOTE: the previous example of truncated math now throws an error with both v4.1.2 and v5.0.0-alpha.8, so big improvement there!
